lib/stages/prepare_build.sh

```diff
@@ -35,7 +35,7 @@ _stage_prepare_build() {
         composer require oomphinc/composer-installers-extender --no-update --working-dir="${DRUPAL_TESTING_DRUPAL_INSTALLATION_DIRECTORY}"
     fi
 
-    composer require drush/drush:"^11.2.0" --no-update --working-dir="${DRUPAL_TESTING_DRUPAL_INSTALLATION_DIRECTORY}"
+    composer require drush/drush:"^12.0.0@stable" --no-update --working-dir="${DRUPAL_TESTING_DRUPAL_INSTALLATION_DIRECTORY}"
```

Comment:
Why do we put the version on this string - wouldn't it be better just to let composer work this out?

Reply:
```suggestion
    composer require drush/drush:"@stable" --no-update --working-dir="${DRUPAL_TESTING_DRUPAL_INSTALLATION_DIRECTORY}"
```

Reply:
I worry about the merge plugin here meaning that we'd not recognise constraints. Hmmm.

Reply:
```suggestion
    composer require drush/drush --no-update --working-dir="${DRUPAL_TESTING_DRUPAL_INSTALLATION_DIRECTORY}"
```
As we have prefer stable the @stable is pointless.